process.md

```diff
@@ -0,0 +1,57 @@
+# Process
+
+We follow the OKR process, see the [2019/2020 Delivery Plan](https://wellcomecloud.sharepoint.com/sites/wc2/DE/RoadMaps/Delivery%20Plans/Delivery%20Plan%202019-20%20Digital%20Engagement.pdf?csf=1&e=WUG2dI&cid=56bdefa5-c5d4-44a1-99e7-96a6975d10fc)
+
+Delivery over process - do not follow these guidelines if they prevent you from delivering value as defined by our objectives.
+
+## Projects
+
+Projects are quarterly iterations of the projects [available in Github](https://github.com/wellcometrust/platform/projects).
+
+Issues are binned or recycled on a quarterly basis.
+
+## Pull Requests (PRs)
+
+All Pull Requests should have in general:
+
+- What is the benefit of this change (and for who)
```

Comment:
nit: s/benefit/outcome?

Reply:
even bigger nit, I'm sorry that I'm like this: s/who/whom

Reply:
I love it. I'm a sucker for nit picks.

Reply:
> nit: s/benefit/outcome?

This is why would should do it, next point it what it is.

Reply:
Ah yeah, I was thinking outcomes in the sense of "user outcomes", this is clearer

Reply:
This is deliberately not a BDD "narrative" because _why_ is more important than _what_.